php-frontend/src/main/java/org/sonar/php/tree/impl/declaration/ParameterListTreeImpl.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010 SonarSource and Akram Ben Aissi
+ * sonarqube@googlegroups.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonar.php.tree.impl.declaration;
+
+import java.util.Iterator;
+
+import org.sonar.php.tree.impl.PHPTree;
+import org.sonar.php.tree.impl.SeparatedList;
+import org.sonar.php.tree.impl.lexical.InternalSyntaxToken;
+import org.sonar.plugins.php.api.tree.Tree;
+import org.sonar.plugins.php.api.tree.declaration.ParameterListTree;
+import org.sonar.plugins.php.api.tree.declaration.ParameterTree;
+import org.sonar.plugins.php.api.tree.lexical.SyntaxToken;
+import org.sonar.plugins.php.api.visitors.TreeVisitor;
+
+import com.google.common.base.Functions;
+import com.google.common.collect.Iterators;
+
+public class ParameterListTreeImpl extends PHPTree implements ParameterListTree {
+
+  private static final Kind KIND = Kind.PARAMETER_LIST;
+  private InternalSyntaxToken openParenthesis;
+  private SeparatedList<ParameterTree> parameters;
+  private InternalSyntaxToken closeParenthesis;
+
+  public ParameterListTreeImpl(
+    InternalSyntaxToken openParenthesis, 
+    SeparatedList<ParameterTree> parameters, 
+    InternalSyntaxToken closeParenthesis
+    ) {
+      this.openParenthesis = openParenthesis;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)